docs/api/createSlice.mdx

```diff
@@ -235,7 +235,7 @@ Result's function `getInitialState` provides access to the initial state value g
 ```ts
 import { createSlice, createAction } from '@reduxjs/toolkit'
 import type { PayloadAction } from '@reduxjs/toolkit'
-import { createStore, combineReducers } from 'redux'
+import { configureStore, combineReducers } from 'redux'
```

Comment:
configureStore needs to be imported from @reduxjs/toolkit instead of the core package

Reply:
Resolved